hera_cal/abscal.py

```diff
@@ -1044,7 +1044,7 @@ def abscal_run(data_files, model_files, pol_select=None, verbose=True, overwrite
             if all_antenna_gains:
                 raise ValueError("can't run gen_phs_cal when all_antenna_gains is True")
             AC.phs_logcal(verbose=verbose)
-            AC.data = apply_gains(AC.data, AC.ant_phi_gain)
+            AC.data = apply_gains(AC.data, AC.ant_phi_gain, gain_convention='divide')
```

Comment:
Isn't this already the default? Why specify it here and not elsewhere in run_abscal?

Reply:
yup, i added it to all `apply_gains` calls